app-misc/crunch/crunch-3.6.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A wordlist generator"
+DESCRIPTION="Crunch is a wordlist generator where you can specify a standard character set \
+or a character set you specify. crunch can generate all possible combinations \
+and permutations."
+HOMEPAGE="https://sourceforge.net/projects/crunch-wordlist/"
+COPYRIGHT="2009, 2010, 2011 bofh28"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/crunch-wordlist/files/crunch-wordlist/crunch-$portVersion.tgz"
+CHECKSUM_SHA256="6a8f6c3c7410cc1930e6854d1dadc6691bfef138760509b33722ff2de133fe55"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	crunch$secondaryArchSuffix = $portVersion
+	cmd:crunch$secondaryArchSuffix
```

Comment:
For your latest comment, no commandSuffix needed, but also no $secondaryArchSuffix, the binary will be installed in $prefix/bin, so:
```suggestion
	cmd:crunch
```